<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KScheduledView</class>
 <widget class="QWidget" name="KScheduledView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_qbuttonNew">
       <property name="text">
        <string>New Scheduled transaction...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_searchWidget">
       <property name="placeholderText">
        <string comment="Placeholdertext in schedulesview">Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KMyMoneyTreeView" name="m_scheduleTree">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyTreeView</class>
   <extends>QTreeView</extends>
   <header>kmymoneytreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
